tools/docker-compose/Dockerfile

```diff
@@ -19,9 +19,7 @@ RUN yum -y update && yum -y install openssh-server ansible mg vim tmux \
   zanata-python-client gettext gcc-c++ libcurl-devel bzip2 \
   rsync
 
-RUN ln -s /usr/bin/python36 /usr/bin/python3
```

Comment:
Looks like there's a new centos7 image as of ~20 days ago that does this now.  We had an old cached one on our zuul nodes.